### DISCUSSION TITLE
plugin does not show up in pro tools latest 2021.6.0

### DISCUSSION THREAD

**Discussion by Jarrodsz** [apohl79/audiogridder#855] [Help]
I installed the plugin in latest pro tools on mac but it won't load up in pro tools.

Is there a solution for this? 
I already removed the aax cache in pro tools and restarted it several times, did reinstall several times

**Comment by Jarrodsz** [comment]
The plugins are in avid active plugins folder

AudioGridder.aaxplugin
AudioGridderInst.aaxplugin
AudioGridderMidi.aaxplugin

But nothing show up when Im trying to add them to the track

**Comment by Jarrodsz** [comment]
Anyone has this problem also?
Higly frustrating... I tried every version of the audiogridder plugin beta intel/univeral. etc nothing works not showing up in pro tools

**Reply by Watto5150** [reply]
I got the Beta 1.2 version working fine so far But im on PC
The folder hierarchy is all wrong with the beta 1.2 version aax plugin the aax just goes to plugins folder, not in folders like the 1.1 version so do this
Install version 1.1 then go to c drive/program files/common files/Avid/audio/plug-ins
And copy the 3 Audiogridder folders and paste them to a folder on your desktop
then uninstall version 1.1 and install beta v1.2
then go back to c drive/program files/common files/Avid/audio/plug-ins
and cut the aax plugin and replace it to the one in folders on your desktop/folder you created/Audiogridder.aaxplugin/contents/x64 paste and over ride original file
and do the the same with instrument and midi aax
then copy the 3 folders back to the plugins folder
do a restart then it works fine
The server needs VST plugins installed on it it dosent scan AAX kind of works similar to Blue Cats Patchworks as a vst wrapper
Tested it with waves V13 plugins so far what i Found so far VST presets not visible But the VST3 presets work 
Noticed minimal latency on default settings with 5 Plugins on the chain
Haven't fully tested it yet but got it working for an hour

**Comment by apohl79** [comment]
As I wrote on the other thread, I can't verify those builds. Try the M1 build of the AG plugins with the M1 version of pro tools. The CPU architectures need to match. In theory the universal build should work as well, as it has both architectures in included, but I don't know if pro tools does.

**Comment by beefybanana** [comment]
I'm having exactly the same issue. Pro Tools 2022.12 on Mac Intel. Installed the plugin and it's in the plugin folder, but it doesn't scan and can't be accessed in Pro Tools. Tried the intel version, and universal, same behavior for both.

Just heard about Audiogridder, and would be really useful for me, so it's a bit of a disappointment to find it doesn't work.
